src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,8 @@
+package duke.command;
+
+public class ByeCommand implements Command {
+    //Custom byeCommand Method to exit Duke
```

Comment:
The comment can be more refined. Actually I am not sure whether you are describing the ByeCommand class or the run method

Reply:
I have modified the comment as appropriate to really emphasise that it exists only to end the Duke program.